src/pages/notice/NoticeList.tsx

```diff
@@ -1,3 +1,204 @@
+    SORT_OPTIONS[0],
+  ); //정렬(드롭다운값)
+  const [filterValues, setFilterValues] = useState<FilterValues>({}); // 상세필터 상태값
+  const [currentPage, setCurrentPage] = useState<number>(1); // 현재 페이지(페이지네이션)
+  const [filterOpen, setFilterOpen] = useState<boolean>(false); // 필터 모달 오픈
+  const [loading, setLoading] = useState(false); // 로딩 에러처리
+  const [error, setError] = useState<string | null>(null); // 로딩 에러처리
+
+  useEffect(() => {
+    setLoading(true);
+    setError(null);
+
+    getNotices({
+      offset: (currentPage - 1) * ITEMS_PER_PAGE,
+      limit: ITEMS_PER_PAGE,
+      address: filterValues.address?.[0],
+      keyword: search,
+      startsAtGte: filterValues.startsAt ?? undefined,
+      hourlyPayGte: filterValues.hourlyPay ?? undefined,
+      sort: sortMap[sort] ?? undefined,
+    })
+      .then((data) => {
+        setAllNotices(data.items.map((item) => item.item));
+        setTotalCount(data.count);
+      })
+      .catch((err) => setError(err.message))
+      .finally(() => setLoading(false));
+  }, [search, sort, filterValues, currentPage]);
+
+  const totalPages = Math.ceil(totalCount / ITEMS_PER_PAGE); // 페이지네이션
+  const appliedFilterCount = countAppliedFilters(filterValues); // 필터 적용
+
+  // 필터 적용
+  const handleApplyFilter = (values: FilterValues) => {
+    setFilterValues(values);
+    setCurrentPage(1);
+  };
+
+  // ===== 렌더 =====
+  if (loading)
+    return (
+      <div className="flex h-500 items-center justify-center text-h3 text-black">
+        잠시만 기다려주세요
+      </div>
+    );
+  if (error)
+    return (
+      <div className="flex h-500 items-center justify-center text-h3 text-red-500">
+        {error}
+      </div>
+    );
+
+  return (
+    <main>
+      {/* 맞춤 공고 */}
+      {!search && (
+        <article className="bg-red-10 px-32 py-60">
+          <section className="mx-auto flex max-w-964 flex-col">
+            <h2 className="mb-32 text-h1 font-bold">맞춤 공고</h2>
+            <div className="flex gap-14 overflow-hidden">
+              {allNotices.slice(0, 3).map((notice) => (
+                <Link
+                  key={notice.id}
+                  to={`/shops/${notice.shop.item.id}/notices/${notice.id}`}
+                  className="block"
+                >
+                  <Post data={notice} />
+                </Link>
+              ))}
+            </div>
+          </section>
+        </article>
+      )}
+
+      {/* 전체 공고 */}
+      <article className="px-32 py-60">
+        <section className="mx-auto flex max-w-full flex-col gap-32 sm:max-w-964">
+          <div className="flex flex-col gap-12 sm:flex-row sm:justify-between">
+            <h2 className="mb-4 text-h1 font-bold sm:mb-0">
+              {search ? (
+                <>
+                  <span className="text-h1 font-bold text-primary">
+                    {search}
+                  </span>
+                  <span className="text-h1 font-bold">에 대한 공고 목록</span>
```

Comment:
```suggestion
                  <span className="text-primary">
                    {search}
                  </span>
                  에 대한 공고 목록
```
💬 이런식으로도 작성 가능할 것 같습니다~ 기본적으로 font 설정들은 상속이 되기 때문에 상위 태그에서 지정해주면 하위에서는 설정하지 않을 수도 있습니다

Reply:
네 알겠습니다!